src/services/project/modules/ProjectCardList.vue

```diff
@@ -359,7 +357,7 @@ export default {
                 else res = await listProjectApi(getParams(id, text), { cancelToken: listProjectToken.token });
                 state.items = res.results;
                 state.totalCount = res.total_count;
-                projectPageState.projectCount = state.totalCount;
+                projectPageStore.projectCount = state.totalCount;
```

Comment:
todo: You have to use `$patch` when modifying state!

```suggestion
                projectPageStore.$patch({ projectCount: state.totalCount });
```